client/src/main/java/org/apache/uniffle/client/impl/ShuffleReadClientImpl.java

```diff
@@ -293,6 +293,9 @@ public void checkProcessedBlockIds() {
 
   @Override
   public void close() {
+    if (sdr != null) {
```

Comment:
Do we need this modification? We have released readBuffer.

Reply:
I think this will not cause any side effects, and it will be more readable? I can revert this if you want.